db/migrate/20240919134424_increase_size_of_id_sample_lims_column_in_sample.rb

```diff
@@ -0,0 +1,9 @@
+class IncreaseSizeOfIdSampleLimsColumnInSample < ActiveRecord::Migration[7.0]
+  def up
+    change_column :sample, :id_sample_lims, :string, limit: 255
+  end
```

Comment:
255 is default limit, so probably no need to specify it, but also I suppose, no harm to do so.